docs/documentation/updating-the-kit.md

```diff
@@ -90,6 +90,26 @@ git checkout master && git merge FETCH_HEAD
 
 You shouldn't have any merge conflicts if you've not changed files outside of the 'app' folder. If you do, there are plenty of ways to fix the conflicts and [github has instructions](https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/) that should be easy to follow if you need a refresher course.
 
+Finally, after updating the kit it's time to start the application just to test that everything went smoothly. In terminal or command prompt type:
+
+```
+npm start
+```
+
+All being well the prototyping kit application will run as normal. If you see an error after updating, there may have been changes to underlying code that require an extra command to be run before you can start the application. Type the following into your terminal or command prompt: 
```

Comment:
I don't think its easy to link to this content as it stands, so yeh just copying and pasting for consistency would be good.

I don't think we need the second instance of:

```
After the kit has started, you should see a message telling you that the kit is running:

Listening on port 3000 url: http://localhost:3000
```

as we've just said that above